superset/databases/filters.py

```diff
@@ -49,7 +49,7 @@ def apply(self, query: Query, value: Any) -> Query:
         filtering.
         """
 
-        if dynamic_filters := current_app.config["EXTRA_DYNAMIC_QUERY_FILTERS"]:
```

Comment:
@butuzov what we currently have seems correctly as this key _should_ always be defined per [here](https://github.com/apache/superset/blob/af577d64b17a9730e28e9021376318326fe31437/superset/config.py#L1676).

Reply:
That seams I was handed extremely customized superset codebase by another team. I will close PR, then. thank you!